src/components/loginForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { FunctionComponent } from "react";
```

Comment:
You cloud import `FC` instead of `FunctionComponent` as it's shorter, clear and has the same purpose